docs/asciidoc/modules/ROOT/pages/graph-updates/graph-refactoring/clone-nodes.adoc

```diff
@@ -32,14 +33,48 @@ image::apoc.refactor.cloneNodes.dataset.png[scaledwidth="100%"]
 ----
 MATCH (f:Foo{name:'Foo'}),(b:Bar{name:'Bar'})
 CALL apoc.refactor.cloneNodes([f,b])
-YIELD input, output
+YIELD input, output, error
 RETURN *
 ----
 
+where `input` is the source node id, `output` is the node cloned and `error` is the possible error message.
+
 If we execute this query, it will result in the following graph:
 
 image::apoc.refactor.cloneNodes.png[scaledwidth="100%"]
 
+In case of error, the message will be thrown in the `error` result, for example with this constraint:
+[source,cypher]
+----
+CREATE CONSTRAINT ON (n:UniqueLabel) ASSERT n.key IS UNIQUE
+----
+
+We can create this node:
+
+[source,cypher]
+----
+CREATE (:UniqueLabel {key: 1})
+----
+
+and then we can execute:
+
+[source,cypher]
+----
+MATCH (n:UniqueLabel) WITH n
+CALL apoc.refactor.cloneNodes([n]) 
+YIELD error, output RETURN error, output
```

Comment:
Should RETURN be on a separate line here for consistency with the other queries?